safe_transaction_service/history/views.py

```diff
@@ -267,6 +268,7 @@ def get(self, request, *args, **kwargs):
         by a delegate). If you need that behaviour to be disabled set the query parameter `trusted=False`
         - Module Transactions for a Safe. `tx_type=MODULE_TRANSACTION`
         - Incoming Transfers of Ether/ERC20 Tokens/ERC721 Tokens. `tx_type=ETHEREUM_TRANSACTION`
+        Ordering_fields: `[created]`  (https://www.django-rest-framework.org/api-guide/filtering/#orderingfilter)
```

Comment:
Nitpick: Since this `ordering_fields` is part of the main framework we're using, and it is used in other parts of the project, I wouldn't include a link to the framework's documentation, what do you think?

Reply:
I included this: `https://www.django-rest-framework.org/api-guide/filtering/#orderingfilter` could be enough?

Reply:
I misunderstanding your message, you mean that we shouldn't include this doc link. I included to add some general samples but I haven't a strong opinion about this then I changed by an example ;)